Week6/Accelerometer.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from 'react';
+import { Accelerometer } from 'expo-sensors';
+import { Subscription } from 'expo-battery';
+
+export function AccelerometerSDK() {
```

Comment:
Since this contains other hooks, this should be a custom hook.  You could name it something like `useAcceleromterSDK`